rm-main/src/logging.rs

```diff
@@ -0,0 +1,21 @@
+use std::fs::{File, OpenOptions};
+
+/// Get an async file writer for the `rustmission.log` file located in XDG state path.
+pub fn get_log_file() -> Option<File> {
```

Comment:
It would be more preferable if it would be moved to rm-config/src/logging.rs, and then its output saved in rm-config/src/lib.rs in the Directories struct. It would be then available as a global variable under the name of CONFIG (which is also defined in rm-config/src/lib.rs)